### ISSUE TITLE
Dumps from v0.25.2 can not be imported to v0.26rc (rc2 and rc3)

### ISSUE THREAD

**Issue by eskombro** [meilisearch/meilisearch#2199]
A dump created by an instance of meilisearch v0.25.2 can not be used as a dump import in v0.26.0. This behavior has been checked with `v0.26.0rc`, `v0.26.0rc2` and `v0.26.0rc3`

**To Reproduce**
1. Run a v0.25.2 meilisearch instance.
2. Index a few thousand documents
3. Call the `/dumps` route and check that the dump status is "done"
4. Stop v0.25.2 instance (make sure to remove `data.ms` folder just to be sure)
5. run a v0.26rc3 instance with the following command:
```bash
./meilisearch --import-dump <path-to-dump>
```
6. Meilisearch throws an error:
```
Error: a character literal was not valid
```

**Expected behavior**
v0.26.0 should run and it should have indexed all of the documents present in the previous instance and therefore in the dump

**Comment by curquiza** [comment]
Thanks @eskombro the core team is working on it, sorry of the bug!

**Comment by curquiza** [comment]
Fixed by #2200
